### PR TITLE
V3 lunatic component types

### PR DESCRIPTION
**Pull request by nsenave**
Mostly Lunatic components have a `"componentType"` property, which is required for Lunatic.

This information is not automatically inserted by Lunatic-Model json serializer => mapped it in every concerned components in Eno.

### CODE REVIEW

File: eno-core/src/main/java/fr/insee/eno/core/converter/LunaticConverter.java [hunk 1]
```diff
@@ -74,10 +74,16 @@ else if (enoObject instanceof TableCell)
 
     private static Object instantiateFrom(SingleResponseQuestion enoQuestion) {
         if (enoQuestion instanceof TextQuestion textQuestion) {
-            if (textQuestion.getMaxLength().intValue() < Constant.LUNATIC_SMALL_TEXT_LIMIT)
-                return new Input();
-            else
-                return new Textarea();
+            if (textQuestion.getMaxLength().intValue() < Constant.LUNATIC_SMALL_TEXT_LIMIT) {
```
Comment (davdarras) [inline eno-core/src/main/java/fr/insee/eno/core/converter/LunaticConverter.java:77]: "else" is devil =) cognitive complexity could be significatively reduced here with early returns

File: eno-core/src/main/java/fr/insee/eno/core/converter/LunaticConverter.java [hunk 2]
```diff
@@ -74,10 +74,16 @@ else if (enoObject instanceof TableCell)
...
+                input.setComponentType(ComponentTypeEnum.INPUT);
+                return input;
+            }
+            else {
+                Textarea textarea = new Textarea();
+                textarea.setComponentType(ComponentTypeEnum.TEXTAREA);
+                return textarea;
+            }
         }
         else if (enoQuestion instanceof NumericQuestion)
```
Comment (davdarras) [inline eno-core/src/main/java/fr/insee/eno/core/converter/LunaticConverter.java:88]: same here

**Review by davdarras — COMMENTED** [review]
Good for me. Maybe refactor LunaticConverter class to avoid so many "else" and reduce cognitive complexity

File: eno-core/src/main/java/fr/insee/eno/core/converter/LunaticConverter.java (hunk 1 above)
Comment (nsenave) [reply eno-core/src/main/java/fr/insee/eno/core/converter/LunaticConverter.java:77]: True. I opened an issue for this https://github.com/InseeFr/Eno/issues/583

File: eno-core/src/main/java/fr/insee/eno/core/converter/LunaticConverter.java (hunk 2 above)
Comment (nsenave) [reply eno-core/src/main/java/fr/insee/eno/core/converter/LunaticConverter.java:88]: https://github.com/InseeFr/Eno/issues/583

**Pull request merged**